packages/pn-commons/package.json

```diff
@@ -5,9 +5,9 @@
   "main": "./src/index.ts",
   "dependencies": {
     "@date-io/date-fns": "^2.15.0",
-    "@mui/icons-material": "5.11.11",
-    "@mui/lab": "5.0.0-alpha.100",
-    "@mui/material": "5.10.9",
+    "@mui/icons-material": "^5.14.3",
+    "@mui/lab": "5.0.0-alpha.140",
+    "@mui/material": "5.14.5",
     "@mui/system": "5.10.8",
     "@mui/x-date-pickers": "5.0.13",
```

Comment:
As before

Reply:
`"@mui/x-date-pickers"` is giving some issues while testing/compiling. Probably that's due to some updates, here are some tests errors of `pn-commons`:
`src/components/CustomDatePicker.tsx:8:10 - error TS2314: Generic type 'DesktopDatePickerProps<TDate>' requires 1 type argument(s).

8   props: DesktopDatePickerProps<DatePickerTypes, DatePickerTypes> &
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/components/CustomDatePicker.tsx:23:34 - error TS7006: Parameter 'view' implicitly has an 'any' type.

23     getViewSwitchingButtonText={(view) =>
                                    ~~~~

src/components/CustomDatePicker.tsx:36:29 - error TS7006: Parameter 'value' implicitly has an 'any' type.

36     getOpenDialogAriaText={(value, utils) => {
                               ~~~~~

src/components/CustomDatePicker.tsx:36:36 - error TS7006: Parameter 'utils' implicitly has an 'any' type.

36     getOpenDialogAriaText={(value, utils) => {
                                      ~~~~~


Found 4 errors in the same file, starting at: src/components/CustomDatePicker.tsx:8`

Reply:
Currently working on it to see how to fully solve the problem. 
It seems that there is a doc for the migration from v5 to v6 -> https://mui.com/x/migration/migration-pickers-v5/

Reply:
It is now full migrated and works properly.